recipe_scrapers/_abstract.py

```diff
@@ -63,50 +63,50 @@ def __init__(
 
     @classmethod
     def host(cls) -> str:
-        """Returns the host domain of the recipe URL"""
+        """Host domain of the recipe URL"""
         raise NotImplementedError("This should be implemented.")
 
     def canonical_url(self):
-        """Returns the canonical_url or original URL of the recipe"""
+        """Canonical_url or original URL of the recipe"""
         canonical_link = self.soup.find("link", {"rel": "canonical", "href": True})
         if canonical_link:
             return urljoin(self.url, canonical_link["href"])
         return self.url
 
     def title(self):
-        """Returns the title of the recipe."""
+        """Title of the recipe."""
         raise NotImplementedError("This should be implemented.")
 
     def category(self):
-        """Returns the category of the recipe."""
+        """Category of the recipe."""
         raise NotImplementedError("This should be implemented.")
 
     def total_time(self):
-        """Returns the total time needed to prepare and cook the recipe in minutes."""
+        """Total time needed to prepare and cook the recipe in minutes."""
         raise NotImplementedError("This should be implemented.")
 
     def cook_time(self):
-        """Returns the cooking time in minutes."""
+        """Cooking time in minutes."""
         raise NotImplementedError("This should be implemented.")
 
     def prep_time(self):
-        """Returns the preparation time in minutes."""
+        """Preparation time in minutes."""
         raise NotImplementedError("This should be implemented.")
 
     def yields(self):
-        """Returns the total servings or items in the recipe."""
+        """Total servings or items in the recipe."""
         raise NotImplementedError("This should be implemented.")
 
     def image(self):
-        """Returns the image of the recipe."""
+        """Image of the recipe."""
```

Comment:
```suggestion
        """An image URL for the recipe."""
```

Reply:
Definitely an improvement here, I'm planning at looking into image retrieval at some point as i've seen a few issues related to it.